src/localize/i18n.ts

```diff
@@ -0,0 +1,31 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
```

Comment:
Shouldn't be needed, see: https://github.com/swissgeol/ngm/blob/a56d2037ce7de748007fc7038c6be7b43ae9cd23/src/i18n.js